Automattic/simplenote-android
# Add encrypted credentials for upcoming screenshot user

### Fix
Adds the credentials to use for the user that will drive the screenshot automation via `.configure`.

### Test
1. `bundle install && bundle exec fastlane run configure_apply`
1. Open Android Studio and make the project
1. Verify that `SCREENSHOT_USER_EMAIL` and `SCREENSHOT_USER_PASSWORD` are present in `BuildConfig.java`

### Review
Only one developer is required to review these changes, but anyone can perform the review. Thanks.

### Release
These changes do not require release notes.
